### PR TITLE
fix: Show default version if ldflag not specified

### PR DESCRIPTION
**Pull request by alexsadliak**
Hey, I implemented a fix, which was discussed in #78 .

I have read the project's contributing guidelines, where the fifth point says that I need to create tests for my changes. However, version command could not be tested without changing the flag parsing in main() method, hence I skipped that part.

Also, the contributing guidelines say about good commit messages, I looked up latest commits in the repo and came up with something similar.

Hope, I will be of any help for you, guys :smile:

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 //VERSION of ergo.
-var VERSION string
+var VERSION = "unofficial build"
```
Comment (cristianoliveira) [inline main.go]: Could we add some comment explaining why it is has this version? 
```
// VERSION of ergo.
// When ergo is build without a proper tag/release it is named as `unofficial version`
// For instance, installing through `go get github.com/cristianoliveira/ergo`
```

Comment (alexsadliak) [reply main.go]: Of course, in my opinion, it is reasonable

**Comment by alexsadliak** [comment]
Hi @cristianoliveira , I have made the requested changes, are they okay?

**Comment by cristianoliveira** [comment]
LGTM. :)

**Pull request merged**